src/module/item/spell/sheet.ts

```diff
@@ -393,6 +381,30 @@ export class SpellSheetPF2e extends ItemSheetPF2e<SpellPF2e> {
         return { id, level, type, base, dataPath: base, system };
     }
 
+    /**
+     * Retrieve the default value for this property.
+     * The default attempts to reuse the most recent heightened value, or what's on the spell, but certain properties
+     * have alternative defaults.
+     */
+    #getDefaultProperty(property: string): unknown {
+        const scaling = this.item.getHeightenLayers().reverse();
+        const baseValue = (() => {
+            for (const entry of [...scaling, { system: this.item.system }]) {
+                if (objectHasKey(entry.system, property)) {
+                    return entry.system[property];
+                }
+            }
+
+            return undefined;
```

Comment:
Any reason this can't be `return null`?

Reply:
Null is theoretically a valid value, but I had to throw an exception instead.